meta/runtime.yml

```diff
@@ -69,461 +69,137 @@ plugin_routing:
         warning_text: You are using an internal name to access the community.network.aruba_config
           modules. This has never been supported or documented, and will stop working
           in community.network 8.0.0.
-    network.avi.avi_actiongroupconfig:
-      redirect: community.network.avi_actiongroupconfig
-      deprecation:
-        removal_version: 8.0.0
-        warning_text: You are using an internal name to access the community.network.avi_actiongroupconfig
-          modules. This has never been supported or documented, and will stop working
-          in community.network 8.0.0.
```

Comment:
No matter what happens (content removed or not, redirects added to your collection, ...), these redirects have to stay.